mods/d2k/rules/infantry.yaml

```diff
@@ -150,7 +150,7 @@ fremen:
 	Cloak:
 		InitialDelay: 85
 		CloakDelay: 85
-		UncloakOn: Attack, Unload, Infiltrate, Demolish, Dock, Damage
+		UncloakOn: Attack, Unload, Infiltrate, Demolish, Dock, Damage, Heal, SelfHeal
```

Comment:
> `SelfHeal` if damage < 0 and caused by self (ex. `SelfHealing`).

So `SelfHealing` provided by the elite rank ([LinkToLine](https://github.com/OpenRA/OpenRA/blob/bleed/mods/d2k/rules/defaults.yaml#L87)) would trigger the `UncloakOn`?

If so I would only add `UncloakOn: Heal`, else Elite Fremen would have a disadvantage compared to "normal" Fremen. Same for Saboteur even it is difficult to get an Elite Saboteur :P